docs/pages/versions/unversioned/config/metro.mdx

```diff
@@ -181,16 +182,140 @@ Expo Metro has _partial_ support for SCSS/SASS.
 
 To setup, install the `sass` package in your project:
 
-```sh
-yarn add sass
-```
+<Terminal cmd={['$ yarn add -D sass']} cmdCopy="yarn add -D sass" />
 
 Then, ensure [CSS is setup](#css) in the `metro.config.js` file.
 
 - When `sass` is installed, then modules without extensions will be resolved in the following order: `scss`, `sass`, `css`.
 - Only use the intended syntax with `sass` files.
 - Importing other files from inside a scss/sass file is not currently supported.
 
+### Tailwind
+
+> Tailwind does not support native platforms, this is web-only.
+
+[Tailwind](https://tailwindcss.com/) can be used with Metro for web, however due to the advanced caching system in Metro, the setup is a little different from the default Tailwind setup. The following files will be modified:
+
+<FileTree
+  files={[
+    'app.json',
+    'package.json',
+    'global.css',
+    'metro.config.js',
+    'tailwind.config.js',
+    'index.js',
+  ]}
+/>
+
+To setup, install the `tailwindcss` package in your project:
+
+<Terminal cmd={['$ yarn add -D tailwindcss@^3.3.1']} cmdCopy="yarn add -D tailwindcss@^3.3.1" />
```

Comment:
```suggestion
<Terminal cmd={['$ yarn add -D tailwindcss@^3.3.1']} />
```

No need for manually add `copyCmd` string until we only have one line command.

Also, is there a reason why we require at least certain version of TW (no lower than 3.3.1)? Can't we just use `latest` or skip the version tag in here?